DAS/das/tests/test_logger.py

```diff
@@ -0,0 +1,36 @@
+from das.utils import logger
```

Comment:
We've got to figure these folders out... Despite this being in a `tests` folder, this is the only thing that's a unit test, the rest I guess are more like testing utilities?

Reply:
Whatever isn't a test inside the `tests` folder should relocated into the utils folder for now I reckon. (different PR)

Reply:
Yeah definitely need to move them into unils